README.md

```diff
@@ -132,7 +145,7 @@ you should generate documentation with all renderers:
 
 ```bash
 $ cd example
-$ yarn
+$ yarn install
 $ yarn docs
```

Comment:
What about removing every `yarn docs:...` for only one `yarn docs:all`?

Reply:
I understand, but I don't think this is useful here in E2E context.

When you update the source code, you want to generate example docs for every renderers (default, docstrap, ...), not only one. 
To be honest, I stopped using `yarn docs:docstrap` or `yarn docs:tui` when `yarn docs:all` was introduced by @silverspectro.

Maybe we can change to this:

```bash
$ cd example
$ yarn install

# generate docs for every renderers
$ yarn docs:all

# or one by one
$ yarn docs
$ yarn docs:docstrap
$ yarn docs:minami
$ yarn docs:tui
```

Reply:
Affirmative. You make a good point. The suggestion is clear as well.